orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/client/SaltConnector.java

```diff
@@ -153,8 +153,12 @@ public GenericResponses action(SaltAction saltAction) {
         return run(null, fun, clientType, clazz, arg);
     }
 
-    @Measure(SaltConnector.class)
     public <T> T run(Target<String> target, String fun, SaltClientType clientType, Class<T> clazz, String... arg) {
+        return run(target, fun, clientType, clazz, null, arg);
```

Comment:
based on description is this mean that the default timeout is 15 secs?

Reply:
The default timeout for salt is 5 sec. Only the network.ipaddrs calls will use 15 sec timeout.